scripts/image-builder/blueprint_v0.0.1.toml

```diff
@@ -0,0 +1,51 @@
+name = "microshift-container"
+
+description = ""
+version = "0.0.1"
+modules = []
+groups = []
+
+# MicroShift and dependencies
+
+[[packages]]
+name = "microshift"
+version = "*"
+
+[[packages]]
+name = "conntrack-tools"
```

Comment:
`conntrack-tools` are a dependency of microshift.rpm now, so this entry should no longer be needed.